com.bitwig.BitwigStudio.yaml

```diff
@@ -25,56 +25,43 @@ finish-args:
   # For desktop notifications
   - --talk-name=org.freedesktop.Notifications
 
-  # Bitwig Studio specific folders
-  - --filesystem=~/.BitwigStudio
-  - --filesystem=~/Bitwig Studio
-
-  # Standard Plugin folders
-  - --filesystem=~/.vst
-  - --filesystem=~/.vst3
-  - --filesystem=/usr/lib64/vst
-  - --filesystem=/usr/lib32/vst
-  - --filesystem=/usr/lib/vst
-  - --filesystem=/usr/local/lib64/vst
-  - --filesystem=/usr/local/lib32/vst
-  - --filesystem=/usr/local/lib/vst
-
-  # Extra plugin folder
-  - --filesystem=~/.u-he
-
-  # Convenient folders
-  - --filesystem=xdg-desktop
-  - --filesystem=xdg-music
-  - --filesystem=xdg-documents
-  - --filesystem=xdg-download
-  - --filesystem=xdg-public-share
-  - --filesystem=xdg-data
-  - --filesystem=xdg-templates # might be needed by plugins
-  - --filesystem=xdg-config    # might be needed by plugins
+  # Give access to HOME
+  - --filesystem=home
 
   # Java stuff
   - --persist=~/.java
 
+  - --env=ALSA_CONFIG_PATH=
+  - --env=VST_PATH=/app/extensions/Plugins/lxvst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '19.08'
+    add-ld-path: lib
+    merge-dirs: lxvst;vst3
+    subdirectories: true
+    no-autodownload: true
+
 modules:
   - name: "bitwig-studio"
     buildsystem: simple
     build-commands:
-      - ar x bitwig-studio-3.2.deb
+      - ar x bitwig-studio-*.deb
```

Comment:
What happens if someone uses the same folder to rebuild the flatpak when there is a new version?

Reply:
it works. The manifest only download one .deb, so there will only be one. The build directory is replace everytime.